cpu/esp32/Makefile.include

```diff
@@ -100,7 +100,6 @@ INCLUDES += -I$(ESP32_SDK_DIR)/components/soc/include
 INCLUDES += -I$(RIOTBOARD)/common/$(CPU)/include
 INCLUDES += -I$(RIOTCPU)/$(CPU)
 
-CFLAGS  += -DSCHED_PRIO_LEVELS=32
```

Comment:
This change isn't required anymore. The `SCHED_PRIO_LEVELS` setting was fixed with PR #12753.